linking/runtime-linking/resolve-function-by-brute-ratel-badger-hash.yml

```diff
@@ -0,0 +1,34 @@
+rule:
+  meta:
+    name: resolve function by Brute Ratel Badger hash
+    namespace: linking/runtime-linking
+    authors:
+      - jakub.jozwiak@mandiant.com
+    description: Custom API hashing algorithm used in Brute Ratel Badger (version 1.3 or higher)
+    scope: function
+    att&ck:
+      - Defense Evasion::Obfuscated Files or Information::Dynamic API Resolution [T1027.007]
+    references:
+      - https://bruteratel.com/release_notes/releases.txt
+    examples:
+      - 64ce9ab801d9bef5284b408c3373dd30ba2dc6952c0950c8049be067b5f24530:0x6DB42430
+  features:
+    - or:
+      - basic block:
+        - and:
+          - mnemonic: add
+          - instruction:
+            - mnemonic: imul
```

Comment:
should we add `mul` here as well?

Reply:
Sure! Added in https://github.com/mandiant/capa-rules/pull/793/commits/20c12b55da587fc8cd72adc08ba5c78c09d6b56c